_data/navbars/security.yml

```diff
@@ -2,31 +2,35 @@ title: "Security"
 path: /security/
 order: 3
 section:
-- title: Adopt a zero trust network model for security
-  path: /security/adopt-zero-trust
-- title: Get started with policy
-  path: /security/get-started
-  section:
-  - title: Calico policy
-    path: /security/calico-policy
-    section:
-    - title: Get started with Calico network policy
-      path: /security/calico-network-policy
-    - title: Get started with Calico network policy for OpenStack
-      path: /security/network-policy-openstack
-  - title: Kubernetes policy
-    path: /security/kubernetes-policy
+  - title: Adopt a zero trust network model for security
+    path: /security/adopt-zero-trust
+  - title: Get started with policy
+    path: /security/get-started
     section:
-    - title: Get started with Kubernetes network policy
-      path: /security/kubernetes-network-policy
-    - title: Kubernetes policy, demo
-      path: /security/tutorials/kubernetes-policy-demo/kubernetes-demo
-    - title: Kubernetes policy, basic tutorial
-      path: /security/tutorials/kubernetes-policy-basic
-    - title: Kubernetes policy, advanced tutorial
-      path: /security/tutorials/kubernetes-policy-advanced
-  - title: Enable default deny 
-    path: /security/kubernetes-default-deny    
+      - title: Calico policy
+        path: /security/calico-policy
+        section:
+          - title: Get started with Calico network policy
+            path: /security/calico-network-policy
+          - title: Get started with Calico network policy for OpenStack
+            path: /security/network-policy-openstack
+          - title: Calico Policy tutorial
```

Comment:
Perhaps the build issue is that this title doesn't match the one in the new file?

Reply:
I messed up while dealing with the merge conflicts! I took the latest stuff from master and put in my tutorial in the _data/navbars/security.yml now it should be working.  I can build the site locally on my machine.